### PR TITLE
feat(windows): convert Keyman for Windows help to Markdown

### PR DESCRIPTION
**Pull request by mcdurdin**
1. This is an initial conversion, so much editing to do.
2. Establishes a build script with pandoc dependency.
3. Works to build both .chm and .md on website.
4. Docbook is no longer required; removed in follow-up PR.

NOTE: Editing of markdown files and content will be a separate PR. This is a baseline only.

The most relevant files to examine here are Makefile and build.sh.

### CODE REVIEW

File: windows/src/desktop/help/about/credits.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Credits
```
Comment (darcywong00) [inline windows/src/desktop/help/about/credits.md:1]: Do the id's need to be preserved?
e.g. `<section id="about_credits">`

File: windows/src/desktop/help/about/license.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# End User Licence and Services Agreement
+
+The MIT License
+
+Copyright (c) 2017 SIL International
```
Comment (darcywong00) [inline windows/src/desktop/help/about/license.md:5]: Does this update to 2020?

File: windows/src/desktop/help/about/credits.md (hunk 1 above)
Comment (darcywong00) [reply windows/src/desktop/help/about/credits.md:1]: nvm, I see the new links in index.md

File: windows/src/desktop/help/about/welcome.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+standards and can be used with all standard hardware keyboards, making
+it the perfect solution for typing in any language on your computer.
+
+A list of features in Keyman is available [here](#about_whatsnew).
+
+![](../desktop_images/osk_keyboard.png)
+
+**Related Topics**
+
+-   [???](#about_whatsnew)
```
Comment (darcywong00) [inline windows/src/desktop/help/about/welcome.md:38]:
```suggestion
-   [What's New](whatsnew]
```

File: windows/src/desktop/help/about/welcome.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+-   [???](#about_requirements)
```
Comment (darcywong00) [inline windows/src/desktop/help/about/welcome.md:40]:
```suggestion
-  [Requirements](requirements)

**Comment by mcdurdin** [comment]
@darcywong00, from the initial comment;

> NOTE: Editing of markdown files and content will be a separate PR. This is a baseline only.

File: windows/src/desktop/help/index.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+* Basic Help
+
+* Advanced Help
+
+* Common Questions
+
+* Troubleshooting
+
+* Contextual Help
```
Comment (darcywong00) [inline windows/src/desktop/help/index.md:15]: Might as well get these starting links in

```suggestion
* [About Keyman](about)

* [Getting Started](start)

* [Basic Help](basic)

* [Advanced Help](advanced)

* [Common Questions](common)

* [Troubleshooting](troubleshooting)

* [Contextual Help](context)

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**